### PR TITLE
Build itzo for ARM

### PR DESCRIPTION
**Pull request by hidalgopl**
~We might want to implement execs endpoints~

### CODE REVIEW

File: pkg/server/exec_arm.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+func (s *Server) runExecCmd(ws *wsstream.WSReadWriter, cmd *exec.Cmd, interactive bool) error {
+	return errors.New("not supported on arm")
+}
+
+func (s *Server) runExecTTY(ws *wsstream.WSReadWriter, cmd *exec.Cmd, interactive bool) error {
+	return errors.New("not supported on arm")
+}
+
```
Comment (ldx) [inline pkg/server/exec_arm.go:42]: Is there anything that's x86-specific in exec.go?

Comment (hidalgopl) [reply pkg/server/exec_arm.go:42]: Downstream dependency: https://github.com/jandre/procfs/issues/11 Do you think this is anything we can easily fix there / in our fork?

Comment (hidalgopl) [reply pkg/server/exec_arm.go:42]: I guess possible solution could be switching to something similar with better support, e.g. https://github.com/prometheus/procfs

Comment (hidalgopl) [reply pkg/server/exec_arm.go]: actually it seems that will be almost a transparent change in terms of API

Comment (ldx) [reply pkg/server/exec_arm.go]: Yeah, that's a lot better than stubbing out the whole file for ARM.

**Comment by hidalgopl** [comment]
@henryprecheur I'm merging this one - it basically changes `/proc` pseudo file parser to one that can be compiled for ARM64; this unblocks us to have itzo ARM64 binary.

**Pull request merged**